harness/determined/transformers/_hf_callback.py

```diff
@@ -60,7 +60,7 @@ def on_log(
             return
         metrics, metric_type = self._get_metrics(logs)
         logger.debug(f"on_log metrics, global_step {state.global_step}", metrics)
-        if metric_type == TRAIN:
+        if metric_type in [TRAIN, TRAIN_AVG]:
```

Comment:
it doesn't seem like `TRAIN` is being used? can we just get rid of it, and name `TRAIN_AVG` to `TRAIN`?